src/pages/ResetPasswordPage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import '../style/App.scss';
+import AppStoreContainer from '../components/app-store-container/AppStoreContainer';
+import ResetPasswordForm from '../components/users/ResetPasswordForm';
+import smilies from '../assets/smilies.png';
+
+const ResetPasswordPage = () => {
+  const [uid, token] = [
+    window.location.href.split('/')[4],
```

Comment:
You can use `URLSearchParams` to get query params, in this way if in the future the type of URL changes you don't have to rewrite this logic

Reply:
You can use useParams hooks too, from react-router-dom.
example: https://reactrouter.com/web/api/Hooks/useparams